templates/Datashades-OpsWorks-CKAN-Instances.cfn.yml.j2

```diff
@@ -166,24 +166,28 @@ Resources:
                 echo '/dev/sdi /mnt/local_data xfs defaults,nofail 0 2' >> /etc/fstab
                 mount -a
               fi
+              if ! (yum install chef); then
+                for i in `seq 1 5`; do
+                  yum install -y libxcrypt-compat "https://packages.chef.io/files/stable/chef/14.15.6/el/7/chef-14.15.6-1.el7.x86_64.rpm" && break
```

Comment:
can you add aws doc's or similar on which version we need to be locked to.

Reply:
I actually set v14 before discovering that we needed `libxcrypt-compat`. It appears that we can use the latest v18 instead.